src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -501,7 +501,7 @@ class KotlinPoetTest {
             .initializer("%S", "a")
             .build())
         .addType(TypeSpec.classBuilder("B")
-            .superclass(ClassName("", "A"))
+            .superclass(ClassName("com.squareup.tacos", "A"))
```

Comment:
nit: `tacosPackage` instead of hardcoded package name?

Reply:
oh nice, done 👍